app/controllers/feedbacks_controller.rb

```diff
@@ -0,0 +1,9 @@
+class FeedbacksController < ApplicationController
```

Comment:
Why a separate controller instead of putting this create method into RevisionFeedbackController?